addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -169,6 +172,20 @@ class AttachmentViewer extends Component {
         }
     }
 
+    /**
+     * Hide some buttons in PDF.js
+     *
+     * @private
+     */
+    _handlePdfButtons() {
+        if (!this.attachmentViewer || !this.attachmentViewer.attachment) {
+            return;
+        }
+        if (this.attachmentViewer.attachment.fileType === 'application/pdf') {
+            hidePDFJSButtons(this.el);
+        }
```

Comment:
It should be possible to target the exact iframe with a `t-ref`, and only making a condition on its existence.

Note that due to the async nature of the rendering, the current conditions can actually become unsync with what was rendered during small periods of time, which could lead to the "No IFRAME found" warning being displayed.
(For example in desktop it happens when leaving the left or right key pressed if there are one pdf and one non-pdf attachment and quickly going from one to the other)

Reply:
@seb-odoo Yes, we are aware of this but I cannot reproduce it in practice. I added two attachements (one image et and one pdf) in the chatter of a sale order. Is that correct?

Reply:
Yes I just went on the runbot of the task at the time, added 1 pdf and 1 image on the same chatter, opened the preview, and then just kept the "left key" pressed to keep changing preview quickly and it generated the message due to missing iframe in the console many times.